engine/src/LogicalFilter.cpp

```diff
@@ -348,7 +348,7 @@ void add_expression_to_plan(blazing_frame & inputs,
 					// get_scalar_from_string(right_operand,inputs.get_column(right_index).dtype());
 					//					right_scalars.push_back(left);
 
-					left_inputs.push_back(SCALAR_INDEX);  //
+					// left_inputs.push_back(SCALAR_INDEX);  //
```

Comment:
why are these commented out?

Reply:
just so the compiler doesn't complain, it doesn't really matter because this file is deleted in the filter migration PR